### PR TITLE
add hip support

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

File: src/dray/CMakeLists.txt [hunk 1]
```diff
@@ -296,6 +296,15 @@ if(ENABLE_CUDA)
   set_source_files_properties(linear_bvh_builder.cpp PROPERTIES COMPILE_FLAGS "-Xptxas -dlcm=cg")
 endif()
 
+
+################################################
+# hip support
+################################################
+if(ENABLE_HIP)
+  list(APPEND dray_thirdparty_libs blt::hip)
+endif()
```
Comment (cyrush) [inline src/dray/CMakeLists.txt:305]: Q: What was the final name of the BLT hip wrapper target?

**Comment by cyrush** [comment]
the final blt target retains the name we were using before

**Comment by cyrush** [comment]
@mclarsen  -- can you make me admin on this repo?

**Review by mclarsen — APPROVED** [review]

**Comment by mclarsen** [comment]
@cyrush You are now an admin.

**Comment by cyrush** [comment]
@mclarsen  thanks!

**Pull request merged**